<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>90</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Opção selecionada no menu:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCores">
    <property name="title">
     <string>Cores</string>
    </property>
    <addaction name="actionVerde"/>
    <addaction name="actionAmarelo"/>
    <addaction name="actionAzul"/>
   </widget>
   <addaction name="menuCores"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionVerde">
   <property name="text">
    <string>Verde</string>
   </property>
  </action>
  <action name="actionAmarelo">
   <property name="text">
    <string>Vermelho</string>
   </property>
  </action>
  <action name="actionAzul">
   <property name="text">
    <string>Azul</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
